src/app/components/courses-container/courses-container.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="courses-container">
+    <div class="courses-heading-container">
+      <div class="courses-heading">{{ courseHeading }}</div>
+      <div>
+        <i class="fa-solid fa-chevron-up fa-rotate-270 slider-navigation-arrow slider-prev-navigation-arrow" (click)="!disablePrevButton && onPrevClick()" [class.disabled]="disablePrevButton"></i>
```

Comment:
do the click functions really work with those flags ?

Reply:
These flags make the click function unexecutable when the total page exceeeds, Chandu